src/modules/epsr/process.cpp

```
@@ -559,7 +576,15 @@ Module::ExecutionResult EPSRModule::process(ModuleContext &moduleContext)
     auto &totalRFactor =
         moduleContext.dissolve().processingModuleData().realise<Data1D>("RFactor", name_, GenericItem::InRestartFileFlag);
     totalRFactor.addPoint(moduleContext.dissolve().iteration(), rFacTot);
-    Messenger::print("Current total R-Factor is {:.5f}.\n", rFacTot);
+    Messenger::print("Current total R-Factor over complete range ({:.5f} to {:.5f}) is {:.5f}.\n", lowerBound, upperBound,
```

Here we report the full range over which the errors were calculated, but I think we need to be a bit more vague since the available data ranges for individual datasets can vary (e.g. x-ray data tends to be over a narrower Q-scale). So, reporting the range here is difficult to do in a meaningful way, and I think we should just report the total error.